exprel/dataset/utils.py

```diff
@@ -3,10 +3,81 @@
 import networkx as nx
 import penman as pn
 import re
-from tuw_nlp.graph.utils import preprocess_node_alto
+from tuw_nlp.graph.utils import preprocess_node_alto, preprocess_edge_alto
 
 
-def amr_pn_to_graph(raw_dl, edge_attr='color', clean_nodes=False):
+def ud_to_graph(sen, edge_attr="color"):
```

Comment:
Maybe this function can later be moved to tuw_nlp/graph

Reply:
or maybe even the whole module...

Reply:
Agree, opened an issue!